crates/solvers/src/api/routes/notify/dto/notification.rs

```diff
@@ -100,47 +102,23 @@ pub struct Notification {
     #[serde_as(as = "Option<DisplayFromStr>")]
     auction_id: Option<i64>,
     solution_id: Option<u64>,
+    #[serde(flatten)]
     kind: Kind,
 }
 
 #[serde_as]
 #[derive(Debug, Deserialize)]
-#[serde(rename_all = "lowercase")]
+#[serde(rename_all = "lowercase", tag = "kind")]
```

Comment:
I could only find where you change what the solver expects but not the same change in how the driver sends the data. Meaning I think this would break the `/notify` endpoint.
I'm not aware of any tests that verify that `/notify` is working so a breaking change might go unnoticed.
That being said it might make sense to not expect a specific format on `/notify`. In a completely colocated world if we were to introduce a new type of notification (or do any breaking change really) the solvers would run into deserialization issues and not get the vital information.

Reply:
The changes are in `crates/driver/src/infra/solver/dto/notification.rs`